backend/src/apiserver/model/run.go

```diff
@@ -73,7 +73,7 @@ var runAPIToModelFieldMap = map[string]string{
 	"created_at":    "CreatedAtInSec",
 	"description":   "Description",
 	"scheduled_at":  "ScheduledAtInSec",
-	"storage_state": "StorageState",
+	"storageState": "StorageState",
```

Comment:
Let's use storage_state here, and fix the case in the proto name instead. It's more consistent, and in line with the style guide for proto field naming.

Reply:
We should, but I'm kind of blocked on your bazel changes since I can't re-generate the client libs with the changes. If you think we'll land that change soon, we can block this PR on it. Otherwise, let's get this merged and refactor it separately?

Reply:
Oh yeah good point. I think the Bazel change is good to go. 
@IronPan do you think you can get to that change soon? If not, maybe let's submit this first and change names later.

Reply:
IMO we desperately need #631.